back/engines/commercial/analysis/config/prompts/claude_toxicity_detection.erb

```diff
@@ -0,0 +1,11 @@
+A human user is in dialogue with an AI. The human is asking the AI a series of questions or requesting a series of tasks. Here is the most recent request from the user:
+<content>
+  <%= text %>
+</content>
```

Comment:
Sounds good, will test this out and compare the results.

Reply:
@kogre The prompt you suggest indeed gives better results https://www.notion.so/citizenlab/Implement-a-ruby-replacement-for-the-inappropriate-content-endpoint-3b863a45c5834a27bcecd00d9507c76d?pvs=4#9e72255e30364b82a0d1c236e9502b86.

In general the LLM is now more sensitive. It flagged the link as spam, because this time it reasons that it is not relevant to the topic of the forum. It also flagged an (arguably) insulting expression, which I also consider to be the better decision.